keps/sig-node/727-resource-metrics-endpoint/README.md

```diff
@@ -202,11 +202,11 @@ Alpha:
 
 Beta:
 
-- [ ] Modify the metrics server to consume the kubelet resource metrics endpoint 3 releases after it is added to the kubelet
+- [X] Modify the metrics server to consume the kubelet resource metrics endpoint 3 releases after it is added to the kubelet
 
 GA:
 
-- [ ] Add node-e2e test to the node conformance tests
+- [X] Add node-e2e test to the node conformance tests
```

Comment:
can you add a link to the test?

Reply:
Actually, the [PR](https://github.com/kubernetes/kubernetes/pull/116897) isn't merged yet. Should I just add the PR link here?

Reply:
Added the link to the test.